src/api/routes/offer.js

```diff
@@ -69,7 +69,8 @@ export default (app) => {
      */
     router.get("/company/:companyId", companyValidators.getOffers, async (req, res, next) => {
         try {
-            const offers = await (new OfferService()).getOffersByCompanyId(req.params.companyId, req.targetOwner, req.hasAdminPrivileges);
+            const offers = await (new OfferService())
+                .getOffersByCompanyId(req.params.companyId, req.targetOwner, req.hasAdminPrivileges, {});
```

Comment:
Shouldnt we have some kind of pagination in here? I believe we discussed in the meeting the user should be able to view all of the company offers by clicking somethign like "More offers". What kind of route parameters would be needed to implement this using the new keyset system? @BrunoRosendo 

But we should do this in a different issue

Reply:
Also, if this doesn't work how it used to work (calling the function without that empty object`{}`, you should add a default value to the function so that you don't have to use an empty object everywhere you need this

Reply:
@DoStini It'd be a lot more simpler, since we wouldn't have search in the company's page. We'd just alter the company offers endpoint to use the same logic as in #212

Reply:
What if we reused the offer search function, and passing the company we want to filter on (like if the user passed in that filter anyway). We would get the "pagination" for free after #212